src/harness/compilerRunner.ts

```diff
@@ -183,8 +183,9 @@ class CompilerBaselineRunner extends RunnerBase {
                                 jsCode += result.declFilesCode[i].code;
                             }
                         }
-
+                        options.optimizationEntrypoint = undefined;
                         let declFileCompilationResult = harnessCompiler.compileDeclarationFiles(toBeCompiled, otherFiles, result, function (settings) {
+                            delete tcSettings["optimizationEntrypoint"];

```

Comment:
DTS files are usually verified using the same compiler settings they were compiled with - at first this seemed like a bad idea when considered alongside `optimizationEntrypoint`, but after considering it there's _probably_ no need. I'll double check.